ndb/noxfile.py

```diff
@@ -141,3 +141,36 @@ def doctest(session):
         get_path("docs", "_build", "doctest"),
     ]
     session.run(*run_args)
+
+
+@nox.session(py=DEFAULT_INTERPRETER)
+def system(session):
+    """Run the system test suite."""
+    system_test_path = os.path.join("tests", "system.py")
+    system_test_folder_path = os.path.join("tests", "system")
+    # Sanity check: Only run tests if the environment variable is set.
+    if not os.environ.get("GOOGLE_APPLICATION_CREDENTIALS", ""):
+        session.skip("Credentials must be set via environment variable")
+
+    system_test_exists = os.path.exists(system_test_path)
+    system_test_folder_exists = os.path.exists(system_test_folder_path)
+    # Sanity check: only run tests if found.
+    if not system_test_exists and not system_test_folder_exists:
+        session.skip("System tests were not found")
+
+    # Use pre-release gRPC for system tests.
+    session.install("--pre", "grpcio")
+
+    # Install all test dependencies, then install this package into the
+    # virtualenv's dist-packages.
+    session.install("mock", "pytest")
```

Comment:
`mock` only needed in 2.7

Reply:
The entire "system" session is copy pasta from datastore.